opencolorio_config_aces/clf/transforms/blackmagic/generate.py

```diff
@@ -0,0 +1,248 @@
+        dest_dir.mkdir()
+
+    # Generate full transform.
+
+    generate_clf(
+        ocio.GroupTransform(
+            transforms=[
+                lct,
+                mtx,
+            ]
+        ),
+        TF_ID_PREFIX
+        + "BlackmagicDesign:Input:BMDFilm_WideGamut_Gen5_to_ACES2065-1"
+        + TF_ID_SUFFIX,
+        "Blackmagic Film Wide Gamut (Gen 5) to ACES2065-1",
+        dest_dir / ("BMDFilm-WideGamut-Gen5_to_ACES2065-1" + CLF_SUFFIX),
+        "Blackmagic Film Wide Gamut (Gen 5)",
+        "ACES2065-1",
+        aces_transform_id,
+    )
+
+    # Generate transform for primaries only.
+
+    generate_clf(
+        ocio.GroupTransform([mtx]),
+        TF_ID_PREFIX
+        + "BlackmagicDesign:Input:Linear_BMD_WideGamut_Gen5_to_ACES2065-1"
+        + TF_ID_SUFFIX,
+        "Linear Blackmagic Wide Gamut (Gen 5) to ACES2065-1",
+        dest_dir / ("Linear-BMD-WideGamut-Gen5_to_ACES2065-1" + CLF_SUFFIX),
+        "Linear Blackmagic Wide Gamut (Gen 5)",
+        "ACES2065-1",
+        None,
+    )
+
+    # Generate named transform for log curve only.
+
+    generate_clf(
+        ocio.GroupTransform([lct]),
+        TF_ID_PREFIX
+        + "BlackmagicDesign:Input:BMDFilm_Gen5_Log_to_Linear"
+        + TF_ID_SUFFIX,
+        "Blackmagic Film (Gen 5) Log to Linear Curve",
+        dest_dir / ("BMDFilm-WideGamut-Gen5-Curve" + CLF_SUFFIX),
+        "Blackmagic Film (Gen 5) Log",
+        "Blackmagic Film (Gen 5) Linear",
+        None,
+    )
+
+
+def generate_clf_davinci():
+    """Make the CLF file for DaVinci Intermediate Wide Gamut."""
+
+    # Based on the document "DaVinci_Resolve_17_Wide_Gamut_Intermediate.pdf"
+    # dated 2021-07-31.  The resulting CLF was reviewed by Blackmagic.
+
+    cut = 0.00262409
+    a = 0.0075
+    b = 7.0
+    c = 0.07329248
+    m = 10.44426855
+
+    BASE = 2.0
+    LIN_SB = cut
+    LOG_SLP = c
+    LOG_OFF = c * b
+    LIN_SLP = 1.0
+    LIN_OFF = a
+
+    # The linear slope that would be calculated by OCIO based on continuity of the
+    # derivatives is 10.444266836 vs. the published value of 10.44426855.  Based on
+    # input from Blackmagic, it is preferable to set the linear slope value explicitly.
+    LINEAR_SLOPE = m
+
+    lct = ocio.LogCameraTransform(
+        base=BASE,
+        linSideBreak=[LIN_SB, LIN_SB, LIN_SB],
+        logSideSlope=[LOG_SLP, LOG_SLP, LOG_SLP],
+        logSideOffset=[LOG_OFF, LOG_OFF, LOG_OFF],
+        linSideSlope=[LIN_SLP, LIN_SLP, LIN_SLP],
+        linSideOffset=[LIN_OFF, LIN_OFF, LIN_OFF],
+        linearSlope=[LINEAR_SLOPE, LINEAR_SLOPE, LINEAR_SLOPE],
+        direction=ocio.TRANSFORM_DIR_INVERSE,
+    )
+
+    mtx = create_conversion_matrix("DaVinci Wide Gamut", "ACES2065-1", "CAT02")
+
+    # This transform is not yet part of aces-dev, but an ID will be needed for AMF.
+    # Proposing the following ID:
+    aces_transform_id = (
+        "urn:ampas:aces:transformId:v1.5:"
+        + "ACEScsc.Academy.DaVinci_Intermediate_WideGamut_to_ACES.a1.v1"
```

Comment:
Python thing but you don't need the `+` here, everything in the parenthesis is considered to be the same string.